src/features/user/Profile/components/RedeemModal.tsx

```diff
@@ -74,23 +77,18 @@ export default function RedeemModal({
   async function validateCode(data: any) {
```

Comment:
We were earlier separating validate and redeem into 2 steps, and are now combining it into one step. Should the `validateCode` function be renamed?